callhandle.py

```diff
@@ -0,0 +1,52 @@
+def handle_call(members, all_flights): # Two string arrays. Members contain all members of one flight, eg {"VIKING 11", "VIKING 12"}. all_flights contains all flights currently active, eg. ({"VIKING 11", "VIKING 12"}, {"DARKSTAR", "PYTHON 11", "PYTHON 21", "VIKING 21", "VIKING 22"})
+    flight_name = _find_flight_name(members)
+    #num_members = len(members)
+    is_same_flight = _check_same_flight(members, flight_name)
+    is_same_flight_element = _check_same_element(members, flight_name)
+    if(is_same_flight_element and is_same_flight_element):
+        flight_number = _find_flight_number(members, flight_name)
+        return flight_name + " " + flight_number + " FLIGHT"
```

Comment:
f-strings are really handy in python.

for example, instead of:

`flight_name + " " + flight_number + " FLIGHT"`

try

`f"{flight_name} {flight_number} FLIGHT`